src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -185,17 +183,17 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
           fieldName: 'consentDescription',
           isResizable: true,
           minWidth: (tokenPresent)
-            ? (delegatedPermissions ? 400 : 800)
+            ? (permissionModeType === PERMISSION_MODE_TYPE.User ? 400 : 800)
```

Comment:
Could pull out as a variable since it's used multiple times
```
const permissionModeTypeIsUser = permissionModeType === PERMISSION_MODE_TYPE.User;
```

Reply:
This is actually what @lhao03 had before, with the variable `delegatedPermissions`, but you mentioned that the syntax was a bit weird. Maybe I misinterpreted that?

Reply:
For the sake of moving this all back to the scope of #16, I am going to assume you meant that the variable name chosen was confusing at representing that purpose, so I am enacting your suggested changes and merging this PR back.

If this wasn't what you meant, let us know in #16 👀